css/CSS2/generated-content/content-001.xht

```diff
@@ -5,7 +5,7 @@
         <link rel="author" title="Microsoft" href="http://www.microsoft.com/" />
         <link rel="help" href="http://www.w3.org/TR/CSS21/generate.html#propdef-content" />
         <link rel="help" href="http://www.w3.org/TR/CSS21/generate.html#content" />
-        <link rel="match" href="../reference/no-red-on-blank-page-ref.xht"/>
+        <link rel="match" href="content-001-ref.html" />
```

Comment:
What's the reason to change this? If I'm not misreading, this is simply just introducing a duplicate reference?

Reply:
Line 18 had an incorrect selector of `#div1`. Since there was no element with id `div1`, this had no effect, but all of the other content-* tests have a border of some sort. This could be changed to remove the styling of a border entirely, but I just chose to correct it for consistency with the other tests.

Reply:
Oh, so it's simply a case of me not being able to read. :)